src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const theadFromTable = table.querySelector('thead');
+
+function sortForTable(tableForSort, tableCellIndex) {
+  if (tableForSort.tagName === 'TABLE') {
+    const tBody = document.querySelector('tbody');
```

Comment:
The `tbody` should be selected relative to `tableForSort`, not the entire document. Use `tableForSort.querySelector('tbody')` to ensure the correct `tbody` is targeted.